examples/integrations/neo4j/neo4j-se/src/test/java/io/helidon/examples/quickstart/se/MainTest.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2020 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.examples.quickstart.se;
+
+import java.util.concurrent.TimeUnit;
+
+import javax.json.JsonArray;
+
+import io.helidon.examples.integrations.neo4j.se.Main;
+import io.helidon.media.jsonp.JsonpSupport;
+import io.helidon.webclient.WebClient;
+import io.helidon.webserver.WebServer;
+
+import org.junit.Ignore;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.neo4j.driver.AuthTokens;
+import org.neo4j.driver.GraphDatabase;
+import org.testcontainers.containers.Neo4jContainer;
+
+/**
+ * Main test class for Neo4j Helidon SE quickstarter.
+ */
+public class MainTest {
+
+    private static Neo4jContainer neo4jContainer;
+
+    private static WebServer webServer;
+    private static WebClient webClient;
+    //private static final JsonBuilderFactory JSON_BUILDER = Json.createBuilderFactory(Collections.emptyMap());
+
+
+    //@BeforeAll Decide if we need testcontainers.
```

Comment:
Why is the test commented out?

Reply:
Should we use testcontainers?

Reply:
@romain-grecourt ?)

Reply:
testcontainers does not support kubernetes, all our CI is running in kubernetes ;  so we cannot use it.

Reply:
Should I just remove these tests? or there is another option?

Reply:
Does it need to be running in a container ? We can have sidecar containers but that would require these test to be running in a separate stage of the pipeline. Can you start neo4j in-memory in the same VM as junit ?

Reply:
Switched to Neo4j test harness. No containers needed anymore.